wordpress-mobile/WordPress-iOS
# [Dark Mode] Crash for unexpected state

Fixes #12519 

This PR fixes a crash that happens when the app changes its app state to `.background`. 
Because the app [is notified that the trait is changed](https://twitter.com/smileyborg/status/1175199999367368704), it reloads the content to update the style. That generates an exception with the mutable attributed string used to generate the post content.

## To test:
• Run this branch and go to the reader
• Open a post and check everything works normally
• Switch user interface style (light to dark or vice versa)
• Put the app in background and check it doesn't crash
• Switch user interface style (light to dark or vice versa) and bring back the app
• Put the app in background again and check it doesn't crash

Update release notes:
